### PR TITLE
Make alias of Range/NdRange in __init__

### PR DESCRIPTION
**Pull request by chudur-budur**
Now instead of doing `from numba_dpex.core.kernel_interface.utils import Range, NdRange` you can import them like `from numba_dpex import Range, NdRange`

- [x] Have you provided a meaningful PR description?
- [x] Have you added a test, reproducer or referred to an issue with a reproducer?
- [x] Have you tested your changes locally for CPU and GPU devices?
- [x] Have you made sure that new changes do not introduce compiler warnings?
- [ ] If this PR is a work in progress, are you filing the PR as a draft?

### CODE REVIEW

File: numba_dpex/examples/kernel/scan.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 import dpnp as np
 
 import numba_dpex as ndpx
```
Comment (adarshyoga) [inline numba_dpex/examples/kernel/scan.py:9]: Better to keep the import statements consistent in all examples. If were doing "import numba_dpex as dpex" in other examples, we should do the same here.

**Review by diptorupd — APPROVED** [review]
LGTM! Please rebase on top of main and merge.

File: numba_dpex/examples/kernel/scan.py (hunk 1 above)
Comment (chudur-budur) [reply numba_dpex/examples/kernel/scan.py:9]: Done.

**Comment by chudur-budur** [comment]
> LGTM! Please rebase on top of main and merge.

Done. Thanks!

**Pull request merged**